source/services/wix/extractors/communities-blog-app/test/index.js

```diff
@@ -14,15 +14,18 @@ const data = [
 				name: 'bob',
 			},
 		],
-		tags: [],
+		tags: {
+			tags: [],
+			metaData: { total: 0 },
+		},
 		categories: [],
 		extracted: {},
 	},
 ];
 data[ 0 ].extracted.posts = data[ 0 ].posts;
 data[ 0 ].extracted.authors = data[ 0 ].authors;
 data[ 0 ].extracted.categories = data[ 0 ].categories;
-data[ 0 ].extracted.tags = data[ 0 ].tags;
+data[ 0 ].extracted.tags = data[ 0 ].tags.tags;
 
 test.each( data )( 'extract posts', async ( testData ) => {
```

Comment:
So my idea was to use multiple data arrays and iterate over them using `test.each()` and the comparison would happen against the extracted property.